crates/forge/src/run_tests/workspace.rs

```diff
@@ -58,6 +58,11 @@ pub async fn run_for_workspace(args: TestArgs) -> Result<ExitStatus> {
         }
     }
 
+    if !args.skip.is_empty() {
+        let skip_patterns = args.skip.join(";");
+        set_forge_test_skip(skip_patterns);
```

Comment:
I'd move the logic for joining to `set_forge_test_skip`. Let's keep the "contract" for the `SNFORGE_TEST_SKIP` format in a relevant function.